plugins/infinispan90/src/main/java/org/radargun/service/Infinispan90HotrodService.java

```diff
@@ -10,9 +12,23 @@
 @Service(doc = Infinispan60HotrodService.SERVICE_DESCRIPTION)
 public class Infinispan90HotrodService extends Infinispan81HotrodService {
 
+   @Property(doc = "Fully specified Marshaller class, if no class is defined, the default Marshaller - JBossMarshaller is used" +
```

Comment:
The two lines of the doc will actually be on a single line. There's a missing space or line separator.